wcfsetup/install/files/acp/templates/trophyAdd.tpl

```diff
@@ -29,12 +29,16 @@
 		});
 		
 		elBySel('input[name=awardAutomatically]').addEventListener('change', function () {
-			if (elBySel('input[name=awardAutomatically]').checked) {
-				elBySel('.conditionSection').style.display = 'block';
-			} 
-			else {
-				elBySel('.conditionSection').style.display = 'none';
-			}
+			var awardAutomatically = elBySel('input[name=awardAutomatically]').checked;
+
+			elBySelAll('.conditionSection', null, function(section) {
```

Comment:
You can simplify this even more:
```js
var awardAutomatically = elBySel('input[name=awardAutomatically]').checked;
elBySelAll('.conditionSection', null, (awardAutomatically ? 'elShow' : 'elHide'))
```
(I've used a variable for readability here.)